windows/src/engine/keyman32/k32_globals.cpp

```diff
@@ -498,7 +498,7 @@ void Globals::PostControllers(UINT msg, WPARAM wParam, LPARAM lParam)
 	 SendDebugMessageFormat(GetFocus(), sdmGlobal, 0, "PostControllers [%x : %x, %x]", msg, wParam, lParam);
 	for(int i = 0; i < f_MaxControllers; i++) {
   	if(!PostMessage(f_ControllerWindow[i], msg, wParam, lParam)) {
-  	  SendDebugMessageFormat(f_ControllerWindow[i], sdmGlobal, 0, "PostControllers[%d] failed with %x", i, GetLastError());
+      DebugLastError("PostMessage");
```

Comment:
This loses the index detail but gets the correct last error. I think that's good enough...